packages/core/src/constants/constants.ts

```diff
@@ -0,0 +1 @@
+export  const MINIMUM_TIME_BETWEEN_ASSERTIONS = 60 * 60; // 1 hour
```

Comment:
This should be `MINIMUM_SECONDS_BETWEEN_ASSERTIONS`, we will get into seconds / ms hell otherwise.

Reply:
I completely agree, this has been renamed!